Makefile

```diff
@@ -10,8 +10,11 @@ default:
 angel:
 	@./mvnw clean test -Dversion.org.springframework.boot=1.2.8.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Angel.SR6
 
+dalston:
+	@./mvnw clean test -Dversion.org.springframework.boot=1.5.9.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Dalston.SR5
+
 edgware:
-	@./mvnw clean test -Dversion.org.springframework.boot=1.5.9.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Edgware.RELEASE
+	@./mvnw clean test -Dversion.org.springframework.boot=1.5.9.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Edgware.SR3
```

Comment:
I think this should be using the same versions as in pom. e.g. SB   1.5.12.RELEASE. 

travis.yml should be also changed accordingly - as default the default is edgware we don't need a separate job for it, but the edgware job should be renamed to dalston

Reply:
I had the same thought about the version alignment, but wasn't sure.
I will fix it and also update `travis.yml`.

Sorry for these noobs mistakes, but I haven't used Travis before :)

Reply:
Updated Travis configuration, can you please take a look?